pep-0698.rst

```diff
@@ -391,7 +390,7 @@ Mark a base class to force explicit overrides on subclasses
 
 We considered including a class decorator ``@require_explicit_overrides``, which
 would have provided a way for base classes to declare that all subclasses must
-use the ``@override`` decorator on method overrides. The overrides library has a
+use the ``@override`` decorator on method overrides. The ``overrides`` library has a
```

Comment:
```suggestion
use the ``@override`` decorator on method overrides. The `Overrides library`_ has a
```

As discussed above, instead of making it a literal (for some reason), why not just link it instead, which is substantially more appropriate and useful to the reader?